core/message-tags-3.3.md

```diff
@@ -80,6 +89,24 @@ Individual tag keys MUST only be used a maximum of once per message. Clients
 receiving messages with more than one occurrence of a tag key SHOULD discard all
 but the final occurrence.
 
+### Empty tagged messages
+
+Servers MUST accept `PRIVMSG` and `NOTICE` events sent with client-only tags even if
```

Comment:
Yes, this is my main concern with using a new verb. Having to enumerate in the spec all the ways in which it needs to be handled to match the way PRIVMSG and NOTICE are handled seems unnecessarily risky.